src/Entity/User.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\UserRepository;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass=UserRepository::class)
+ */
+class User
+{
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $casUid;
+
+    /**
+     * @ORM\Column(type="integer", nullable=true)
+     */
+    private $userId;
+
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private $ldapUNGUid;
```

Comment:
Is this a uuid ?

Reply:
It represents the uid field/username in the ung ldap, which is the same as the utt ldap, so I don't think that the ramsey/uuid is useful here ?

Reply:
Oh okay, because if it was a uuid, the package I suggested tends to improve db performance for UUIDs, at least AFAIK

Reply:
@larueli just a heads up — the same feature was just released as part of Symfony 5.2

https://symfony.com/blog/new-in-symfony-5-2-doctrine-types-for-uuid-and-ulid